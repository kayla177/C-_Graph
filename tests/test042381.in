ENTITY 9BAOZFFK name_9BAO conference
ENTITY 06LTRWVF name_06LT author
ENTITY X4WFGGS3 name_X4WF author
RELATIONSHIP 9BAOZFFK collaborated_with 06LTRWVF 3.2
PRINT 9BAOZFFK
HIGHEST
EXIT
