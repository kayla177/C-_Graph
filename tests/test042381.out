success
success
success
success
06LTRWVF
9BAOZFFK 06LTRWVF 3.2
